File: DS-Algo-with-Java/Lecture-9/Area.java
import java.util.Scanner;

class Test{
    int length;
    int breadth;
    public Test(int l, int b){
        length = l;
        breadth = b;
    }
    public int getArea(){
        return length*breadth;
    }
}


public class Area {
    public static void main(String[] args){
        try (Scanner sc = new Scanner(System.in)) {
            int l,b;

            System.out.print("Enter length: ");
            l = sc.nextInt();
            System.out.print("Enter breadth: ");
            b = sc.nextInt();

            Test obj = new Test(l,b);

            System.out.println("Area: " + obj.getArea());
        }
    }
}

File: DS-Algo-with-Python/Lecture-9/Area.py
class Area:
    length = 0
    breadth = 0
    
    def __init__(self, l, b):
        self.length = l
        self.breadth = b
    
    def getArea(self):
        return self.length * self.breadth
    

l = int(input("Enter length: "))
b = int(input("Enter breadth: "))

obj = Area(l, b)
print(f"Area: {obj.getArea()}")
